components/dashboard/src/components/podkit/breadcrumbs/BreadcrumbNav.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { LinkButton } from "@podkit/buttons/LinkButton";
+import { cn } from "@podkit/lib/cn";
+import { ChevronLeft } from "lucide-react";
+import type { FC } from "react";
+import { MiddleDot } from "../../typography/MiddleDot";
+
+interface BreadcrumbPageNavProps {
+    /**
+     * The title of the current page.
+     */
+    pageTitle: string;
+    /**
+     * The description of the current page.
+     */
+    pageDescription?: string;
+    /**
+     * The link to the previous page.
+     */
+    backLink?: string;
+    className?: string;
+}
+
+export const BreadcrumbNav: FC<BreadcrumbPageNavProps> = ({ pageTitle, pageDescription, backLink, className }) => {
+    return (
+        <section className={cn("flex flex-row items-center justify-start gap-2 w-full py-4 app-container", className)}>
+            {backLink && (
+                <LinkButton
+                    variant={"ghost"}
+                    className="py-1 px-0 hover:bg-gray-200 dark:hover:bg-gray-800 dark:hover:text-gray-200"
+                    href={backLink}
+                >
+                    <ChevronLeft size={24} />
+                </LinkButton>
+            )}
+            <h1 className="text-lg font-bold">{pageTitle}</h1>
```

Comment:
could we use one of the `<Heading* />` typography components here? Since it's `text-lg` it would map to `<Heading3>`. If we wanted to make things more semantically correct, I think we'd want this to actually be an `h1` like you have here. I suppose that's where we could adopt the `asChild` pattern radix uses. Maybe like this?


```jsx
<Heading3 asChild>
  <h1>{pageTitle}
</Heading3>
```

Feels kinda weird though, but better than not being able to use our typography components (that should handle all the dark mode/ light mode and font sizings for us). Thoughts?